algorithms/kernel/dbscan/oneapi/cl_kernels/dbscan_cl_kernels.cl

```diff
@@ -0,0 +1,190 @@
+/* file: dbscan_cl_kernels.cl */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of DBSCAN OpenCL kernels.
+//--
+*/
+
+#ifndef __DBSCAN_CL_KERNELS_CL__
+#define __DBSCAN_CL_KERNELS_CL__
+
+#define DECLARE_SOURCE(name, src) static const char * name = #src;
+
+DECLARE_SOURCE(
+    dbscan_cl_kernels,
+
+    algorithmFPType pow_by_mult(algorithmFPType value, int power) {
+        algorithmFPType result = 1.0;
+        for (int i = 0; i < power; i++)
+        {
+            result *= value;
+        }
+        return result;
+    }
```

Comment:
Mixed codestyle: after function header curly brace on the same line, after a cycle - on the next line